layers/meta-resin-iot2000/recipes-core/images/resin-image.bbappend

```diff
@@ -0,0 +1,13 @@
+IMAGE_FSTYPES_append = " resinos-img hostapp-ext4"
```

Comment:
This should only append "resinos-img". Why the need to explicitly also append "hostapp-ext4" ?